api/segments/serializers.py

```diff
@@ -41,6 +44,7 @@ class Meta:
 
 class SegmentSerializer(serializers.ModelSerializer):
     rules = RuleSerializer(many=True)
+    metadata = MetadataSerializer(required=False, many=True)
```

Comment:
I'm keen for this not to be included on any serializers that are used for list views, which I _think_ this is.

Reply:
I think that this was answered here:  https://github.com/Flagsmith/flagsmith/pull/3315